### DISCUSSION TITLE
Import data from excel to mindmap

### DISCUSSION THREAD

**Discussion by biswajit-goswami** [freeplane/freeplane#1152] [Help]
Has anyone tried or succeeded in importing excel data into mind map.
Eg 
![image](https://github.com/freeplane/freeplane/assets/11291623/41e969d9-8379-4835-9b95-415eb6a1d264)

can it be imported into a mindmap as follows:
 
![image](https://github.com/freeplane/freeplane/assets/11291623/b25fd647-4ca3-4297-940b-edf5c8e9d49d)

**Comment by Antonimo-Dev** [comment]
I would also like to know if this is possible without complex scripting.

**Comment by amariusz** [comment]
For complex cases it's probably not possible. But for the one from the screenshot (single word per cell) it probably is:

1. select range in spreadsheet and copy to clipboard
2. create new node, press f2 to edit
3. paste all content in single node
4. edit->node code -> split node (you should have as many nodes as rows)
5. select all newly created nodes
6. edit->node core->split with 12 words in line

**Comment by UGLmain** [comment]
Hi amariusz,

That is very helpful, thanks. I'll experiment with the split functions.

**Comment by Illegit** [comment]
Hi All, I'm used to create gantt in excel using data in mind maps: I export a branch as xtml file, than I copy the sheet contents in an excel sheet.... just for your info.
